sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java

```diff
@@ -719,6 +723,42 @@ public boolean doesExternalTokenExist(String externalToken, String personUuid) {
 				.and(cb, cb.equal(personRoot.get(Person.EXTERNAL_TOKEN), externalToken), cb.notEqual(personRoot.get(Person.UUID), personUuid)));
 	}
 
+	@Override
+	public long setMissingGeoCoordinates(boolean overwriteExistingCoordinates) {
+		long changedPersons = 0;
+
+		List<PersonIndexDto> indexList = getIndexList(null, null, null, null); // this is automatically filtered by the users jurisdiction
```

Comment:
What exactly is the purpose of this code? You don't do anything with the index list afterwards anyway

Reply:
I use the indexlist to get the Uuids for `personsList`

Reply:
Ah yeah, sorry, didn't catch that!